NYPLCardCreator/BusinessLogic/ValidateAddressResponse.swift

```diff
@@ -2,50 +2,38 @@ import Foundation
 
 final class ValidateAddressResponse {  
   enum Response {
-    case validAddress(message: String, address: Address, cardType: CardType)
-    case alternativeAddresses(message: String, addressTuples: [(Address, CardType)])
+    case validAddress(address: Address, cardType: CardType)
```

Comment:
not opposed to, but I was wondering why you removed the `message` from here but left it in the other cases...

Reply:
The `validations/address` endpoint does not return a message if the address is valid, thats why I only removed it from the `validAddress` case